CONTRIBUTING.md

```diff
@@ -129,6 +129,20 @@ When you have set up your fork of a repository that you want to contribute to an
 4. **Validating and verifying the PR:** 
    - Reviewers can review the pull request on the GitHub website or fetch the PR using the _Fetch GitHub PR_ menu entry in the _Git Repositories_ view inside the Eclipse IDE.
    - Every PR is automatically verified to check that contributors have a [valid Eclipse Contributor Agreement (ECA)](#setting-up-your-eclipse-and-github-account).
+   ### How does a valid PR look like? (Checklist)
```

Comment:
Slight spelling change here

How does a valid PR look?

or

What does a valid PR look like?

Reply:
Done in 465d396b762c1eaf426470b920d807eb1113408b

Reply:
Yes, the grammar on the original is not correct.  Both suggestions are improvements, though I like the second one better.

Reply:
Thank you @jonahgraham and @merks , I used the second one.